massa-versioning/src/versioning.rs

```diff
@@ -878,9 +873,10 @@ impl MipStoreRaw {
     fn update_network_version_stats(
         &mut self,
         slot_timestamp: MassaTime,
-        network_versions: Option<(u32, u32)>,
+        network_versions: Option<(u32, Option<u32>)>,
     ) {
-        if let Some((_current_network_version, announced_network_version)) = network_versions {
+        if let Some((_current_network_version, Some(announced_network_version))) = network_versions
+        {
```

Comment:
I'm going a bit fast on my review, but if announced_network_version is None, I don't think you're updating the stats? We still want to!

Reply:
Done